### PR TITLE
Update install guide 5.0.1, add upgrading details

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

- Adds a note about project versioning
- Adds 5.0.0+ upgrading instructions in the changelog and upgrade guide
- Updates install instructions to 5.0.1

## Motivation and Context

Release of 5.0.1

### CODE REVIEW

File: content/sensu-go/5.0/installation/upgrade.md [hunk 1]
```diff
@@ -11,6 +11,35 @@ menu:
...
+
+{{< highlight shell >}}
+# Restart the Sensu agent
+sudo service sensu-agent restart
+
+# Restart the Sensu backend
+sudo service sensu-agent restart
+{{< /highlight >}}
+
+You can verify the verson using the Sensu command line tools.
```
Comment (apaskulin) [inline content/sensu-go/5.0/installation/upgrade.md]:
```suggestion
You can verify the version using the Sensu command line tools.
```

File: content/sensu-go/5.0/installation/upgrade.md [hunk 2]
```diff
@@ -11,6 +11,35 @@ menu:
...
+To upgrade to the latest version of Sensu Go from version 5.0.0 or later, first [install the latest packages][23].
+
+Then restart the services.
+
+{{< highlight shell >}}
+# Restart the Sensu agent
+sudo service sensu-agent restart
+
+# Restart the Sensu backend
+sudo service sensu-agent restart
```
Comment (apaskulin) [inline content/sensu-go/5.0/installation/upgrade.md]:
```suggestion
sudo service sensu-backend restart
```

File: content/sensu-go/5.0/release-notes.md [hunk 3]
```diff
@@ -9,24 +9,46 @@ aliases:
...
+To upgrade to the latest version of Sensu Go from version 5.0.0 or later, first [install the latest packages][8].
+
+Then restart the services.
+
+{{< highlight shell >}}
+# Restart the Sensu agent
+sudo service sensu-agent restart
+
+# Restart the Sensu backend
+sudo service sensu-agent restart
```
Comment (apaskulin) [inline content/sensu-go/5.0/release-notes.md]:
```suggestion
sudo service sensu-backend restart
```

File: content/sensu-go/5.0/installation/upgrade.md [hunk 4]
```diff
@@ -11,6 +11,29 @@ menu:
...
+
+{{< highlight shell >}}
+# Restart the Sensu agent
+sudo service sensu-agent restart
+
+# Restart the Sensu backend
+sudo service sensu-backend restart
+{{< /highlight >}}
+
+For systems using `systemd`, you may need to run `sudo systemctl daemon-reload` after restarting the services.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/installation/upgrade.md]:
```suggestion
For systems using `systemd`, you should run `sudo systemctl daemon-reload` before restarting the services.
```

File: content/sensu-go/5.0/installation/upgrade.md [hunk 5]
```diff
@@ -11,6 +11,29 @@ menu:
...
+
+You can use the `version` command to verify the version using the `sensu-agent`, `sensu-backend`, and `sensuctl` tools. For example: `sensu-backend version`.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/installation/upgrade.md]:
```suggestion
You can use the `version` command to determine the installed version using the `sensu-agent`, `sensu-backend`, and `sensuctl` tools. For example: `sensu-backend version`.
```

File: content/sensu-go/5.0/release-notes.md [hunk 6]
```diff
@@ -9,24 +9,48 @@ aliases:
...
+
+{{< highlight shell >}}
+# Restart the Sensu agent
+sudo service sensu-agent restart
+
+# Restart the Sensu backend
+sudo service sensu-backend restart
+{{< /highlight >}}
+
+For systems using `systemd`, you may need to run `sudo systemctl daemon-reload` after restarting the services.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/release-notes.md]:
```suggestion
For systems using `systemd`, you should run `sudo systemctl daemon-reload` before restarting the services.
```

File: content/sensu-go/5.0/release-notes.md [hunk 7]
```diff
@@ -9,24 +9,48 @@ aliases:
...
+
+You can use the `version` command to verify the version using the `sensu-agent`, `sensu-backend`, and `sensuctl` tools. For example: `sensu-backend version`.
```
Comment (cwjohnston) [inline content/sensu-go/5.0/release-notes.md]:
```suggestion
You can use the `version` command to determine the installed version using the `sensu-agent`, `sensu-backend`, and `sensuctl` tools. For example: `sensu-backend version`.
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Just a couple of suggestions and otherwise looks great!

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**